content/en/docs/setup/install/multicluster/index.md

```diff
@@ -0,0 +1,724 @@
+apiVersion: install.istio.io/v1alpha1
+kind: IstioOperator
+spec:
+  values:
+    global:
+      meshID: BEDROCK
+      multiCluster:
+        clusterName: BARNEY
+      network: EAST
+      meshNetworks:
+        EAST:
+          endpoints:
+          - fromRegistry: BARNEY
+          - fromRegistry: FRED
+          gateways:
+          - registryServiceName: istio-eastwestgateway.istio-system.svc.cluster.local
+            port: 15443
+EOF
+{{< /text >}}
+
+### Enable Endpoint Discovery
+
+Install a remote secret in `BARNEY` that provides access to `FRED`’s API server.
+
+{{< text bash >}}
+$ istioctl x create-remote-secret \
+    --context=${CTX_FRED} \
+    --name=FRED | \
+    kubectl apply -f - --context=${CTX_BARNEY}
+{{< /text >}}
+
+Install a remote secret in `FRED` that provides access to `BARNEY`’s API server.
+
+{{< text bash >}}
+$ istioctl x create-remote-secret \
+    --context=${CTX_BARNEY} \
+    --name=BARNEY | \
+    kubectl apply -f - --context=${CTX_FRED}
+{{< /text >}}
+
+{{< /tab >}}
+
+{{< tab name="Multi-Primary, Multi-Network" category-value="multi-primary-multi-network" >}}
+
+The following steps will install the Istio control plane on both `FRED` and
+`BARNEY`, making each a {{< gloss >}}primary cluster{{< /gloss >}}. `FRED` is
+on the `WEST` network, while `BARNEY` is on the `EAST` network.
+
+Both `FRED` and `BARNEY` observe the API Servers in each cluster for endpoints.
+Service workloads across cluster boundaries communicate indirectly, via
+dedicated gateways for [east-west](https://en.wikipedia.org/wiki/East-west_traffic)
+traffic.
+
+{{< image width="75%"
+    link="multi-primary-multi-network.svg"
+    caption="Multiple primary clusters on separate networks"
+    >}}
+
+### Configure `FRED` as a primary with services exposed
+
+Install Istio on `FRED`:
+
+{{< text bash >}}
+$ istioctl --context=${CTX_FRED} -f - <<EOF
+apiVersion: install.istio.io/v1alpha1
+kind: IstioOperator
+spec:
+  values:
+    global:
+      meshID: BEDROCK
+      multiCluster:
+        clusterName: FRED
+      network: WEST
+      meshNetworks:
+        WEST:
+          endpoints:
+          - fromRegistry: FRED
+          gateways:
+          - registryServiceName: istio-eastwestgateway.istio-system.svc.cluster.local
+            port: 15443
+        EAST:
+          endpoints:
+          - fromRegistry: BARNEY
+          gateways:
+          - registryServiceName: istio-eastwestgateway.istio-system.svc.cluster.local
+            port: 15443
+EOF
+{{< /text >}}
+
+Install a gateway in `FRED` that is dedicated to
+[east-west](https://en.wikipedia.org/wiki/East-west_traffic) traffic. By
+default, this gateway will be public on the Internet. Production systems may
+require additional access restrictions (e.g. via firewall rules) to prevent
+external attacks. Check with your cloud vendor to see what options are
+available.
+
+{{< text bash >}}
+$ CLUSTER=FRED NETWORK=WEST \
+    samples/multicluster/gen-eastwest-gateway.sh | \
```

Comment:
Same here, I'd prefer putting in a file and then having them kubectl apply it so they know they can save that file into their source control system.